File: library/src/main/java/com/gazman/db/ErrorAnalytics.java
package com.gazman.db;

import com.google.firebase.crashlytics.FirebaseCrashlytics;

class ErrorAnalytics {
    public static void logException(Exception e) {
        if (BuildConfig.DEBUG) {
            e.printStackTrace();
        }
        FirebaseCrashlytics.getInstance().recordException(e);
    }
}

File: src/main/java/com/gazman/db/ErrorAnalytics.kt
package com.gazman.db

import com.google.firebase.crashlytics.BuildConfig
import com.google.firebase.crashlytics.FirebaseCrashlytics

internal object ErrorAnalytics {
    fun logException(e: Exception) {
        if (BuildConfig.DEBUG) {
            e.printStackTrace()
        }
        FirebaseCrashlytics.getInstance().recordException(e)
    }
}